backend/main.py

```
@@ -1,26 +1,7 @@
-  playtime.ping_session(app_name, instance_id)
-
-def OnAppStop(app_name: str, instance_id: str):
-  print(f"Non-steam app {app_name} stopped, ending session...")
-  playtime.stop_session(app_name, instance_id)
-
-def GetPlaytime(app_name: str):
-  timings = playtime.get_playtime(app_name)
-  return json.dumps(timings, default=helpers.json_dumps_stringify)
-
-# ===== Plugin ===== #
+from rpc import *
```

Ruff (0.11.9)

`from rpc import *` used; unable to detect undefined names

(F403)